src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -38,10 +38,11 @@ HARDWARE_INTRINSIC(Sve,           LoadVector,
 // ***************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
 // Special intrinsics that are generated during importing or lowering
 
+HARDWARE_INTRINSIC(Sve,           CreateTrueMaskAll,                                                -1,     -1,      false, {INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue,      INS_sve_ptrue},   HW_Category_Helper,           	  HW_Flag_Scalable|HW_Flag_ReturnsPerElementMask)
```

Comment:
I think this one was also more correct in its previous position. We generally try to keep these alphabetical by name as it makes searching/inserting values simpler and would allow us to more easily optimize this to a binary search in the future if needed.